src/main/java/me/botsko/prism/database/sql/SqlSelectQueryBuilder.java

```diff
@@ -626,7 +627,13 @@ public QueryResult executeSelect(TimeTaken eventTimer) {
                     }
 
                     // data
-                    baseHandler.deserialize(itemMetadata);
+                    try {
+                        baseHandler.deserialize(extraData);
+                    }catch (JsonSyntaxException e) {
+                        if(Prism.isDebug()) {
```

Comment:
Codacy found an issue: [WhitespaceAround: 'if' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)